zulipterminal/model.py

```diff
@@ -327,6 +329,29 @@ def exception_safe_result(self, future: 'Future[bool]') -> bool:
         except zulip.ZulipError:
             return False
 
+    def fetch_all_topics(self, workers: int) -> None:
+        """
+        Distribute stream ids across threads in order to fetch
+        topics concurrently.
+        """
+        with ThreadPoolExecutor(max_workers=workers) as executor:
+            list_of_streams = list(self.stream_dict.keys())
+            thread_objects = {}  # type: Dict[int, Future[bool]]
```

Comment:
This statement isn't necessary any more, but you can move the type hint to the end of the next statement instead.

Reply:
Done :+1:. Note that for space constraint, I had to move this a bit.

Reply:
Updated :+1: